client/src/app/core/core-services/autoupdate.service.ts

```diff
@@ -19,21 +20,28 @@ export interface HasFields {
     fields: Fields;
 }
 
+/**
+ * Whether a relation requests a list of view-models that updates the whole view-model-store of one repo.
+ */
+export interface CanBeFullListUpdate {
+    isFullList?: boolean;
+}
+
 export interface ModelRequest extends HasFields {
     ids: number[];
     collection: string;
 }
 
-export interface GenericRelationFieldDecriptor extends HasFields {
+export interface GenericRelationFieldDecriptor extends HasFields, CanBeFullListUpdate {
```

Comment:
The client-internal have nothing to do with the autoupdate service. You are misusing these interfaces to store some information about the data. If I read the code from `getDescriptorForCollection` right, you also just look at the first relations, but nested relations do not work right?

If you want, I can modify the request builder to return a list of collections that can be full-updated.